packages/compass-collections-ddl/src/components/create-collection-modal/create-collection-modal.jsx

```diff
@@ -119,12 +135,26 @@ class CreateCollectionModal extends PureComponent {
   renderCappedSize() {
     if (this.props.isCapped) {
       return (
-        <div className={classnames(styles['create-collection-modal-is-capped-wrapper'])}>
-          <ModalInput
-            id="capped-size-value"
-            name="Maximum Size (Bytes)"
+        <div className="form-group">
+          <TextInput
+            label="size"
+            type="number"
+            description="Maximum size in bytes for the capped collection."
+            onChange={this.onCappedSizeChange}
             value={this.props.cappedSize}
-            onChangeHandler={this.onCappedSizeChange} />
+          />
+        </div>
+      );
+    }
+  }
+
+
+  renderTimeSeries() {
+    if (this.props.isTimeSeries) {
+      return (
+        <div className={styles['create-collection-modal-is-time-series-wrapper']}>
+          <TimeSeries changeTimeSeriesOption={this.props.changeTimeSeriesOption}
+            timeSeries={this.props.collation}/>
```

Comment:
Should this be `timeSeries={this.props.timeseries}`?

Reply:
Yeah .. nice catch, wondering how this causes no issue 😕

Reply:
wait a minute .. that prop is useless i think, unless we would ever do an "edit" dialog, cause leafygreen fields don't need the prop value to change content while you type. i can probably just ditch it entirely.

Reply:
Ok i've removed it, let me know if you see issues with that.